utils/installer/install.ps1

```diff
@@ -0,0 +1,136 @@
+# set script variables
+$XDG_CONFIG_HOME="$HOME\.config"
+$XDG_DATA_HOME="$HOME\.local\share"
+$XDG_CACHE_HOME=$env:TEMP
+
+# Set Variable to master if not set differently
+if ($null -eq $LVBRANCH) {
+    $LVBRANCH = "master"
+}
+
+$ErrorActionPreference = "Stop" # exit when command fails
+
+function mkdirOnDemand($path) {
+	if ((Test-Path $path) -eq $false) {
+		mkdir $path
```

Comment:
I think it's better to use the full command instead of the alias

```pwsh
New-Item -ItemType Directory ...
```

It will also allow add exception handling if needed.